### ISSUE TITLE
[Suggestion] Disentangle monitors

### ISSUE THREAD

**Issue by darkanx** [Shaosil/LethalCompanyMods-GeneralImprovements#51]
A feature to make the main console and terminal consoles act independently, like in this mod: https://thunderstore.io/c/lethal-company/p/Zaggy1024/TwoRadarMaps/   (left/right switching does not work if this mod is used)

Thank you for all your work on this amazing mod!

**Comment by NilaierMusic** [comment]
Yeah, i think [TwoRadarMaps](https://github.com/Zaggy1024/LC_TwoRadarMaps/) is also incompatible with GeneralImprovements, at least when you just use two of them together. So, incorporating [TwoRadarMaps mod's](https://github.com/Zaggy1024/LC_TwoRadarMaps/) feature directly into GeneralImprovements would be quite neat.
